### PR TITLE
Use the result of the scheduler to determine state result

### PR DESCRIPTION
**Pull request by Ishadijcks**
Related to https://github.com/project-march/march-iv/pull/295

### CODE REVIEW

File: march_state_machine/src/march_state_machine/states/FeedbackActionState.py [hunk 1]
```diff
@@ -1,11 +1,21 @@
...
 import rospy
+from actionlib_msgs.msg import GoalStatus
+
+
+def action_result_cb(userdata, status, result):
+
+    if status == GoalStatus.SUCCEEDED:
+        return 'succeeded'
+    else:
+        return 'aborted'
```
Comment (martijnvandermarel) [inline march_state_machine/src/march_state_machine/states/FeedbackActionState.py]: How/where is this return value used?

Comment (Ishadijcks) [reply march_state_machine/src/march_state_machine/states/FeedbackActionState.py]: It is used in the `SimpleActionState`

Comment (Ishadijcks) [reply march_state_machine/src/march_state_machine/states/FeedbackActionState.py]: So it becomes the result of the `FeedbackActionState` which is then used with `        result = FeedbackActionState.execute(self, ud)`

**Comment by Ishadijcks** [comment]
Yes

**Pull request merged**